examples/frontend/src/frontend/core.cljs

```diff
@@ -0,0 +1,50 @@
+(ns frontend.core
+    (:require [reagent.core :as r]
+              [reitit.core :as rc]
+              [reitit.frontend :as rf]
+              [reitit.frontend.history :as rfh]
+              [reitit.coercion.schema :as rs]))
+
+(def router (atom nil))
+
+(defn home-page []
+  [:div
+   [:h2 "Welcome to frontend"]
+   [:div [:a {:href (rfh/href @router ::about)} "go to about page"]]])
+
+(defn about-page []
+  [:div
+   [:h2 "About frontend"]
+   [:a {:href "http://google.com"} "external link"]
+   [:div [:a {:href (rfh/href @router ::frontpage)} "go to the home page"]]])
+
+(defonce match (r/atom nil))
+
+(defn current-page []
+  [:div
+   (if @match
+     [:div [(:view (:data @match))]])
+   (pr-str @match)])
+
+(def routes
```

Comment:
looks complex. would something like this work:

```clj
(def routes
  (reitit.core/router
    [["/"
      {:name ::frontpage
       :view home-page}]
     ["/about"
      {:name ::about
       :view about-page}]]))
```

Reply:
No.